addons/web/static/src/js/core/session.js

```diff
@@ -301,9 +301,6 @@ var Session = core.Class.extend(mixins.EventDispatcherMixin, {
         var self = this;
         options = _.clone(options || {});
         options.headers = _.extend({}, options.headers);
-        if (odoo.debug) {
-            options.headers["X-Debug-Mode"] = $.deparam($.param.querystring()).debug;
```

Comment:
Just gone ? Not needed anymore ?

Reply:
This was done to keep the debug state during RPC's, but since now the debug mode is stored in session and thus persistant, this is not needed anymore.